test/specs/scenarios/accessibility.js

```diff
@@ -0,0 +1,186 @@
+    })
+
+    // Document Selector
+    it('should verify accessibility for the document selector screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      runAccessibilityTest(driver)
+    })
+
+    //Document Upload
+    it('should verify accessibility for the uploader screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for the document upload confirmation screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+  
+    //   documentUpload.getUploadInput()
+    //   documentUpload.upload('passport.jpg')
+    //   runAccessibilityTest(driver)
+    // })
+
+    //Face
+    it('should verify accessibility for the take a selfie screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector,`?language=${lang}&async=false`)
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      runAccessibilityTest(driver)
+    })
+  
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for the selfie confirmation screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector,`?language=${lang}&async=false`)
+    //   uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+    //   camera.takeSelfie()
+    //   confirm.waitForElementToBeLocated(confirm.confirmBtnSelector)
+    //   runAccessibilityTest(driver)
+    // })
+  
+    it('should verify accessibility for liveness intro screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector,`?language=${lang}&liveness=true`)
+      driver.executeScript('window.navigator.mediaDevices.enumerateDevices = () => Promise.resolve([{ kind: "video" }])')
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      livenessIntro.verifyUIElementsOnTheLivenessIntroScreen(copy)
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify accessibility for camera permission screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector,`?language=${lang}&liveness=true`)
+      driver.executeScript('window.navigator.mediaDevices.enumerateDevices = () => Promise.resolve([{ kind: "video" }])')
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      livenessIntro.verifyUIElementsOnTheLivenessIntroScreen(copy)
+      livenessIntro.clickOnContinueButton()
+      runAccessibilityTest(driver)
+    })
+
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for liveness recording and liveness confirmation screens', async () => {
```

Comment:
Would be nice to open a ticket for the color contrast issue

Reply:
Will do. Also will include `should verify accessibility for the selfie confirmation screen` as this fails for the same reason.

Reply:
Created: CX-4214, also added to the next web platform grooming doc.